src/site/paragraphs/linkTeaser.drupal.liquid

```diff
@@ -25,9 +25,9 @@
     {% endif %}
 {% endif %}
 <li {% if parentFieldName === 'field_spokes' %}class="hub-page-link-list__item"{% endif %}>
-    <a href="{{linkTeaser.fieldLink.url.path}}" {% if linkTeaser.options["target"] %}target="{{ linkTeaser.options["target"] }}"{% endif %} onClick="recordEvent({ event: 'nav-linkslist' });">
-        {% if linkTeaser.fieldLink.title != empty %}
-            <{{ header }} class="{{ headerClass }}">{{ linkTeaser.fieldLink.title }}</{{ header }}>
+    <a href="{{linkTeaser.fieldLink.0.url.path}}" {% if linkTeaser.fieldLink.0.options["target"] %}target="{{ linkTeaser.fieldLink.0.options["target"] }}"{% endif %} onClick="recordEvent({ event: 'nav-linkslist' });">
+        {% if linkTeaser.fieldLink.0.title != empty %}
+            <{{ header }} class="{{ headerClass }}">{{ linkTeaser.fieldLink.0.title }}</{{ header }}>
```

Comment:
Is this temporary while waiting for a CMS fix? It seems weird to have a field just called `0`.

Reply:
Yeah it was temporary, the CMS is generated some empty content in this entities. Refactored to just loop through and print the non-empty ones tho.